typings/webmidi.d.ts

```diff
@@ -0,0 +1,195 @@
+// Type definitions for Web MIDI API 2.0
+// Project: http://www.w3.org/TR/webmidi/
+// Definitions by: six a <https://github.com/lostfictions>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+interface Navigator {
```

Comment:
These `d.ts` files were copied over from `@types` to satisfy the type-checker. They can be removed at some point.